contracts/finality/src/state/public_randomness.rs

```diff
@@ -16,23 +21,23 @@ const PUB_RAND_COMMITS: Map<(&str, u64), PubRandCommit> = Map::new("pub_rand_com
 /// values
 #[cw_serde]
 pub struct PubRandCommit {
-    /// `start_height` is the height of the first commitment
+    /// The height of the first commitment
     pub start_height: u64,
-    /// `num_pub_rand` is the number of committed public randomness
+    /// The amount of committed public randomness
     pub num_pub_rand: u64,
-    /// `height` defines the height that the commit was submitted
-    pub height: u64,
-    /// `commitment` is the value of the commitment.
+    /// The epoch number of Babylon when the commit was submitted
+    pub epoch: u64,
```

Comment:
```suggestion
    pub babylon_genesis_epoch: u64,
```

Reply:
Hmm I'm not sure if we should have the term "genesis" in the code base

- The term refers to a specific network, i.e., Babylon Genesis mainnet, but the contract can be used in any network
- Babylon codebase does not refer to "genesis" anywhere

So maybe `babylon_epoch` is more precise?